CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added a log collection guide ([#579](https://github.com/opensearch-project/opensearch-py/pull/579))
 - Added GHA release ([#614](https://github.com/opensearch-project/opensearch-py/pull/614))
 ### Changed
+- `AsyncOpensearch` can now be re-used after calling `close()` ([#639](https://github.com/opensearch-project/opensearch-py/pull/639))
```

Comment:
Nit: I think this is a bug, so I would move this to a fix rather than something changed, "Fix reuse of `AsyncOpenSearch` after calling `close`." Note the spelling of OpenSearch ;)